docs/backends.md

```diff
@@ -456,4 +456,59 @@ metadata:
 type: Opaque
 stringData:
   password: <parent | jsonPath {.child}>
-```
\ No newline at end of file
+```
+
+### Yandex Cloud Lockbox
+##### YCL Authentication
+Refer to the [IAM overview](https://cloud.yandex.com/en/docs/iam/concepts/) for yandex cloud APIs authorization.
+
+These are the parameters for YCL:
+```
+AVP_TYPE: yandexcloudlockbox
+AVP_YCL_SERVICE_ACCOUNT_ID: Service account ID
```

Comment:
Just one thing to mention - if there is a standard set of env variables that CLI tools (or the go module being used) for Yandex cloud expect to use, I'd prefer that we document those are accepted for AVP and read them in using the code here: https://github.com/argoproj-labs/argocd-vault-plugin/blob/main/pkg/config/config.go#L42 and https://github.com/argoproj-labs/argocd-vault-plugin/blob/main/pkg/config/config.go#L219. The thinking is we want AVP to follow the conventions of other tools in a backends ecosystem to make it predictable. 

But if not, this is perfectly acceptable.

Reply:
Sorry guys, I've thoroughly researched the documentation, there is no standard set of env variables for CLI tools and Go-SDK.